app/slackimport.go

```diff
@@ -553,12 +553,63 @@ func SlackConvertChannelMentions(channels []SlackChannel, posts map[string][]Sla
 }
 
 func SlackConvertPostsMarkup(posts map[string][]SlackPost) map[string][]SlackPost {
-	// Convert URLs in Slack's format to Markdown format.
-	regex := regexp.MustCompile(`<([^|<>]+)\|([^|<>]+)>`)
+	regexReplaceAllString := []struct {
+		regex *regexp.Regexp
+		rpl   string
+	}{
+		// URL
+		{
+			regexp.MustCompile(`<([^|<>]+)\|([^|<>]+)>`),
+			"[$2]($1)",
+		},
+		// bold
+		{
+			regexp.MustCompile(`\*([^*]+)\*`),
+			"*$0*",
```

Comment:
Also, if the character after the first * and/or before the second * is whitespace, the regex should not match. e.g. `* hello world *` would not be in bold.